content/nginxaas-azure/faq.md

```diff
@@ -79,7 +79,7 @@ Common questions about F5 NGINX as a Service for Azure (NGINXaaS).
 
 ### Does NGINXaaS support multiple public IPs, a mix of public and private IPs?
 
-- NGINXaaS supports one public or private IP per deployment. NGINXaaS doesn't support a mix of public and private IPs at this time.
+- NGINXaaS supports one public or private IP per deployment. A single NGINXaaS deployment doesn't support a mix of public and private IPs.  We recommend defining separate deployments.
```

Comment:
nit: _recommendation_ may not be what you want here. Given that the mixed IP scenario does not work, we may just want to say _If you want a mix of public and private IP deployments, consider creating one per IP visibility type_. Us recommending something might make the user think that going the other way is an option, which is not going to work here.